README.md

```diff
@@ -41,12 +48,14 @@ For details on contributions we accept and the process for contributing, see our
 
 ## License
 
-Copyright (c) 2019 Blues Inc. Released under the MIT license. See
+Copyright (c) 2025 Blues Inc. Released under the MIT license. See
 [LICENSE](LICENSE) for details.
 
 [blues]: https://blues.com
 [notehub]: https://notehub.io
 [note-c]: https://github.com/blues/note-c
+[note-arduino]: https://dev.blues.io/tools-and-sdks/firmware-libraries/arduino-library/
```

Comment:
The URL is inconsistent with the URL above

Reply:
it's wireless.dev.blues.com.io

Reply:
It’s because we don’t have note-c documentation on dev.blues.com, which we probably should. I’ll update the note-c link to https://blues.github.io/note-c/ though because we do have that.